### PR TITLE
Added TravisCl

### PR DESCRIPTION
**Pull request by darshil**
As per GCI 2016 Task: https://codein.withgoogle.com/tasks/5189718031466496/?sp-is_beginner=False

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+gem 'github-pages'
+gem 'html-proofer'
```
Comment (jayvdb) [inline Gemfile]: Missing EOL at EOF

**Review by jayvdb — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # gci16.fossasia.org
+[![Build Status](https://travis-ci.org/darshilpatel/gci16.fossasia.org.svg)](https://travis-ci.org/darshilpatel/gci16.fossasia.org)
```
Comment (jayvdb) [inline README.md]: Wrong URL here.

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by darshil** [comment]
@jayvdb Can you please check now?

**Comment by jayvdb** [comment]
Still missing a EOL at EOF

**Comment by darshil** [comment]
@jayvdb How can I fix this? I added a new line at the end of the file.

**Comment by darshil** [comment]
@daminisatya Can you please review this?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+language: ruby
+rvm:
+  - 2.3.3
+install: 
+  - bundle install
+script:
+  - bundle exec jekyll build
+branches:
+  only:
+    - gh-pages
```
Comment (daminisatya) [inline .travis.yml]: Please leave a line after this. :)

**Comment by daminisatya** [comment]
@darshilpatel Just leave a line at the end of the file ( in .travis.yml file) then it's good to go.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+language: ruby
+rvm:
+  - 2.3.3
+install: 
+  - bundle install
+script:
+  - bundle exec jekyll build
```
Comment (jayvdb) [inline .travis.yml:7]: Is this needed?  This isnt a Jekyll site yet

**Comment by darshil** [comment]
@jayvdb @daminisatya Could you please check now?

**Review by daminisatya — APPROVED** [review]
Looks good to me! Good job @darshilpatel :) So regarding the jekyll build @jayvdb True, so it's your call to whether have it or not. But yes it is required.

**Comment by darshilpatel** [comment]
@daminisatya @jayvdb Thanks! Is there anything else I need to do, to get it merged?

**Comment by darshil** [comment]
@codethejason Ready to be merged

**Comment by darshil** [comment]
capture
@codethejason An admin will have to enable the Travis Cl repository.
Link: https://travis-ci.org/fossasia/gci16.fossasia.org

**Pull request merged**